src/store/modules/user/actions.ts

```diff
@@ -99,11 +99,11 @@ const actions: ActionTree<UserState, RootState> = {
   },
 
   // Set User Instance Url
-  setUserInstanceUrl ({ state, commit }, payload){
+  setUserInstanceUrl ({ commit }, payload){
     commit(types.USER_INSTANCE_URL_UPDATED, payload)
   },
 
-  async getEComStores({ commit }, payload) {
+  async getEComStores(payload) {
```

Comment:
I think this will cause abrupt behaviour of this action. If we are not doing anything related to state we should rather create service